### DISCUSSION TITLE
Welcome to Ender 2 Pro Discussions!

### DISCUSSION THREAD

**Discussion by The-300lb-Gorilla** [The-300lb-Gorilla/Ender-2-Pro-v427#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by The-300lb-Gorilla** [comment]
Hello all. Welcome. Pull up a seat and stay a while. No Hate, No politics, No Trolling, Or I will boot you. Just try to get along.
Again, all are welcome, even non Ender 2 Pro users.

**Reply by hejiayv** [reply]
Hello, can you still send the source code of the Ender2por motherboard (HC460F2 CPU)? I searched for a long time and could not find the open source code, which caused me to be unable to modify the firmware until I saw your comment on "https://github.com/CrealityOfficial/Ender-2-Pro/issues/20#issuecomment-1568591078" gave me hope, but since I am in China, I did not find a way to contact you, if you still have the firmware you can email me, I would thank you very much if I could, my email address is "hejiayv0127@163.com" but my location is in China, I don't know if you can email me directly, so if you want to send me, can you reply to this comment by the way 


（Since my English is poor, all my English comes from translation software）
Below is my motherboard and chip:
![9d6d49f1a32af7f3185dd9b12dcd17b](https://github.com/The-300lb-Gorilla/Ender-2-Pro-v427/assets/91043095/bd7daf64-b259-439b-84e8-50a51a61284f)
![fedc9cd41db2f3851edbc4a32893732](https://github.com/The-300lb-Gorilla/Ender-2-Pro-v427/assets/91043095/540d7df2-3d26-44f0-a4ec-e393b3b172c4)
